week3/homework/src/index.js

```diff
@@ -1,3 +1,150 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+const app = express();
+const fs = require('fs');
+const uuidv4 = require('uuid/v4');
+const port = 3041;
+
+app.use(express.json());
+
+function readTodoFiles() {
+  return new Promise(function(resolve, reject) {
+    fs.readFile('todo.json', 'utf8', (err, data) => {
```

Comment:
instead of this long function you could require it, `.json` files can be required! as:
```javascript
const todos = require('todos')
```